core/src/main/java/com/datastax/oss/driver/internal/core/os/Native.java

```diff
@@ -34,172 +24,172 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(Native.class);
 
+  private static class ImplLoader {
+
+    public NativeImpl load() {
+      try {
+        return new JnrNativeImpl();
+      } catch (Throwable t) {
+        LOG.info(
+            "Unable to load JNR native implementation.  This could be normal if JNR is excluded from the classpath",
+            t);
+        return new EmptyNativeImpl();
```

Comment:
Manual testing confirmed that (a) I get the error message above and (b) other driver functions appear to work fine (without native support) if JNR is excluded.